sapling-dev-server/Dockerfile

```diff
@@ -21,6 +21,20 @@ RUN npm config set unsafe-perm true
 
 COPY . .
 
+# Hack to add register-login and profiles saplings until there's a distribution
+# strategy for saplings
+RUN git clone https://github.com/hyperledger/grid
```

Comment:
We should hack this in reverse, so that the official code is here and grid is dependent on splinter.